client/index.html

```diff
@@ -8,7 +8,16 @@
 </head>
 <body>
   <script>
-    deeplink.createBrowser('com.android.chrome');
+	var ua = navigator.userAgent;
+	if(/Android/.test(ua)) {
```

Comment:
How about making some utility functions for this checking?
For example..
isAndroid(), isChromiumWebView, isChromeBrowser, isSamsungBrowser.